### PR TITLE
feat: allow AlignmentMode to be used on the command line

### PR DESCRIPTION
**Pull request by nh13**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,9 +15,9 @@ exclude = [".gitignore", ".github"]
 [features]
 phylogeny = ["petgraph"]
 
-
 [dependencies]
 serde = { version = "^1", optional = true, features=["derive"] }
+clap = { version = "4.2.7", optional = true, features = ["derive"] }
```
Comment (nh13) [inline Cargo.toml]:
```suggestion
clap = { version = ">=3.2.0", optional = true, features = ["derive"] }
```

File: src/alignment.rs [hunk 2]
```diff
@@ -38,20 +40,15 @@ pub enum AlignmentOperation {
 ///
 /// The default alignment mode is Global.
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+#[derive(Default, Debug, PartialEq, Eq, Copy, Clone, ValueEnum)]
```
Comment (pmarks) [inline src/alignment.rs]: I think the `ValueEnum` derive needs to be conditional similar to `Serialize`?

Comment (nh13) [inline src/alignment.rs]:
```suggestion
#[cfg_attr(feature = "clap", derive(ValueEnum))]
#[derive(Default, Debug, PartialEq, Eq, Copy, Clone)]
```

**Review by pmarks — APPROVED** [review]
nice, thanks!

**Comment by nh13** [comment]
@pmarks thank-you for your patience as I come up to speed on rust!

**Pull request merged**